array-destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,22 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+function showPeopleBelongIn(array, houseName) {
```

Comment:
Probably worth using a more descriptive name than `array` as it will be easier for people to understand what's going on.

For instance
```js
function showPeopleBelongIn(people, houseName) { }
```

Reply:
`array` changed to `people` and `filteredArray` changed to `peopleInHouse`.
Thanks.